src/utilities/util_write.c

```diff
@@ -16,10 +16,10 @@ static int
 usage(void)
```

Comment:
I guess there is nowhere else to place this, but using *write* to remove is not at all intuitive.
I'd suggest a separate command, to help avoid accidental misuse, but I don't have a sense as to whether the additional work required is merited.

Reply:
I thought about that previously as well. The _write_ command actually has already supported "adding new records" before this PR implicitly (if the key provided is new). That gives me a reason to add the "removing an existing record" to the _write_ command as a command option. Does it make sense to treat _write_ as a general term like _update_?

Reply:
I think it is grey. I don't think "adding a new record" is a good analog, but I do see "allow overwrite" which is destructive, and remove might be thought of as "overwrite with nothing". To me it's a stretch and not intuitive, but given the additional work that would be needed to add an entire command this seems sensible.